### ISSUE TITLE
Does secretflow support secret indexing now?

### ISSUE THREAD

**Issue by linzzzzzz** [secretflow/secretflow#1265]
### Issue Type

Feature Request

### Source

binary

### Secretflow Version

1.4.0b0

### OS Platform and Distribution

Linux

### Python version

3.8.19

### What happend and What you expected to happen.

```shell
From the issue (https://github.com/secretflow/spu/issues/116), it was said that secret indexing was being worked on. My question is whether secret indexing supported now? If yes, from which version?
```

### Reproduction code to reproduce the issue.

```shell
From the issue (https://github.com/secretflow/spu/issues/116), it was said that secret indexing was being worked on. My question is whether secret indexing supported now? If yes, from which version?
```

**Comment by anakinxc** [comment]
Please check comments in https://github.com/secretflow/spu/issues/116

**Comment by anakinxc** [comment]
> Which comment reflect the latest status? This below comment said set reveal_secret_indices as true, however I didn't see reveal_secret_indices in the SPU [documentation](https://www.secretflow.org.cn/en/docs/spu/0.9.0b1/reference/runtime_config#runtimeconfig). Also, is reveal_secret_indices still required?
> 
> [secretflow/spu#116 (comment)](https://github.com/secretflow/spu/issues/116#issuecomment-1471238017)

 The https://github.com/secretflow/spu/issues/116#issuecomment-1487891165 mentioned it was fixed in 0.3.2b9.

Also in changelog [here](https://github.com/secretflow/spu/blob/main/CHANGELOG.md#20230406)

`reveal_secret_indices` was a temporary solution before secret indexing, and was removed after.
